doc/source/getting-started-linux-tutorial.rst

```diff
@@ -14,33 +14,35 @@ Volatility3 does not provide the ability to acquire memory.  Below are some exam
 Be aware that LiME raw format is not supported by volatility3, the padded or lime option should be used instead. `This issue contains further information <https://github.com/504ensicsLabs/LiME/issues/111>`_.
 
 Procedure to create symbol tables for linux
---------------------------------------------
+-------------------------------------------
 
-To create a symbol table please refer to :ref:`symbol-tables:Mac or Linux symbol tables`.
-
-.. tip:: It may be possible to locate pre-made ISF files from the `Linux ISF Server <https://isf-server.techanarchy.net/>`_ ,
```

Comment:
Hi @kevthehermit, This link is not working,
This was a really good project and contributed a lot.
Do you have any plans to keep it maintaining?

Reply:
We probably also want to mention @Abyss-W4tcher 's repo (I think we're just waiting to get that moved into a volatilityfoundation github before we turn it on for all volatility users).

Reply:
Ref: https://github.com/Abyss-W4tcher/volatility3-symbols